### PR TITLE
Variables forders added. Structure orginized.

### PR DESCRIPTION
**Pull request by AndresEstebanGonzalez**

### CODE REVIEW

File: style.scss [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
+  flex-direction: column;
+  background-color: $yellow;
+
+  .top-sector {
+    @include flexCenter;
+
+    width: 100%;
+    height: 10%;
+    .app-title {
+    }
```
Comment (nathan-hyan) [inline style.scss:29]: Delete empty class

File: style.scss [hunk 2]
```diff
@@ -1,6 +1,52 @@
...
+      background-color: $light-green;
+    }
+  }
+  .bottom-sector {
+    @include flexCenter;
+
+    width: 100%;
+    height: 20%;
+    .color-btn {
+    }
```
Comment (nathan-hyan) [inline style.scss:50]: Same here

**Review by nathan-hyan — COMMENTED** [review]
Some comments!

File: style.scss (hunk 1 above)
Comment (AndresEstebanGonzalez) [reply style.scss:29]: Justo son unos detalles que tengo que modificar once este listo el .js

File: style.scss (hunk 2 above)
Comment (AndresEstebanGonzalez) [reply style.scss:50]: Same here.

**Review by nathan-hyan — APPROVED** [review]

**Pull request merged**